src/modules/site-v2/templates/_scripts/utils.js

```diff
@@ -108,6 +105,23 @@ function create_node(html) {
 }
 
 
+function save_svg(selector, filename=null) {
+  const svg_el = document.querySelector(selector);
+  const data = (new XMLSerializer()).serializeToString(svg_el);
+  const svg_blob = new Blob([data], {type: 'image/svg+xml;charset=utf-8'});
+  const url = URL.createObjectURL(svg_blob);
+  download_from_href(url, filename || 'download.svg');
+}
+
+
+function download_from_href(href, filename) {
+  const a = document.createElement('a');
+  a.href     = href;
+  a.download = filename || 'download';
+  a.click();
```

Comment:
should re remove the `a` element from the DOM once the download has been initiated?